sdk/rms_sdk/UnitTests/platform_ut/SConscript

```diff
@@ -63,31 +64,29 @@ libs = [
       'Qt5Network' + lib_suffix,
       'Qt5Xml' + lib_suffix,
       'Qt5XmlPatterns' + lib_suffix,
-      'rms',
       googletest_lib,
 ]
 
 if platform == 'win32':
-  rmsplatform_env.Append(LINKFLAGS=Split('/NOLOGO /DYNAMICBASE /NXCOMPAT /SUBSYSTEM:CONSOLE'))
-  if configuration == 'debug':
-    rmsplatform_env.Append(LINKFLAGS=Split('/DEBUG'))
-  elif configuration == 'release':
-    rmsplatform_env.Append(LINKFLAGS=Split('/INCREMENTAL:NO'))
-  lib_path += [
-      '#third_party/lib/eay',
-  ]
   libs += [      
       'libeay32',
       'ssleay32',
       'Advapi32',
       'Gdi32',
       'User32',
       'dnsapi',
+      'rms', 
   ]  
```

Comment:
may be import rms_lib and use it instead of referencing rms by just name?

Reply:
If yes, then this may be applied to all projects.

---
In reply to: [140046758](https://github.com/AzureAD/rms-sdk-for-cpp/pull/162#discussion_r140046758) [](ancestors = 140046758)

Reply:
Removes rms, it was not needed. We are referencing the objects instead